src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -6,12 +6,13 @@
 
```

Comment:
Why do it in the contextpopup directive? Having it in the popup directive alone should be enough. No?